data/core/units/saurians/Javelineer.cfg

```diff
@@ -0,0 +1,98 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Javelineer
+    name= _ "female^Saurian Javelineer"
+    race=lizard
+    gender=female
+    image="units/saurians/javelineer/javelineer.png"
+    profile="portraits/saurians/skirmisher.webp"
+    hitpoints=50
+    movement_type=lizard
+    movement=7
+    experience=150
+    level=3
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=47
+    usage=archer
+    description= _ "After years of combat practice, saurians warriors that are the most talented in the skill of spear throwing might advance to the ranks of the javelineers. With better equipment and exceptional aim, saurian javelineers are not to be underestimated."
+    die_sound=hiss-die.wav
+    [attack]
+        name=spear
+        description=_"spear"
+        range=melee
+        type=pierce
+        damage=8
+        number=3
+        icon=attacks/spear.png
+    [/attack]
+    [attack]
+        name=spear
+        description=_"spear"
+        range=ranged
+        type=pierce
+        damage=13
+        number=3
+        icon=attacks/spear-thrown.png
+        [specials]
+            {WEAPON_SPECIAL_MARKSMAN}
```

Comment:
I'm not sure this fits for saurians, but maybe there's some reasoning for it?

Reply:
Guess I wanted to make the final upgrade more distinctive. On second thought, I agree it doesn't fit thematically. Do you think 14x3 will be appropriate damage without the ability?

Reply:
Removed the ability.